From 96f009221620e148c55d1a50f7aa6d06f9940ad9 Mon Sep 17 00:00:00 2001
From: Reini Urban <rurban@cpanel.net>
Date: Fri, 4 Apr 2014 11:31:33 -0500
Subject: [PATCH 2/5] 0.54_01: import Fcntl constants for flock

Bareword "LOCK_EX" not allowed while "strict subs" in use at ../blib/lib/Inline.pm line 861
---
 C/Makefile.PL | 4 ++--
 Inline.pm     | 3 ++-
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git C/Makefile.PL C/Makefile.PL
index 03306ee..214b7bd 100644
--- C/Makefile.PL
+++ C/Makefile.PL
@@ -36,8 +36,8 @@ Inline::C is packaged with Inline.pm because it is the most commonly used
 Inline Language Support Module (ILSM).
 
 See also: Inline::ASM, ::Awk, ::BC, ::Basic, ::Befunge, ::CPP (C++), ::CPR,
-          ::Foo, ::Guile, ::Java, ::Octave, ::PERL, ::Python, ::Ruby, ::TT,
-          ::Tcl and ::WebChat.
+          ::Filters, ::Foo, ::Guile, ::Java, ::Octave, ::PERL, ::Python, ::Ruby,
+          ::TT, ::Tcl and ::WebChat.
 
 Config.pm indicates that your version of Perl was built with this C compiler:
 
diff --git Inline.pm Inline.pm
index 062b29f..dc0d504 100644
--- Inline.pm
+++ Inline.pm
@@ -2,7 +2,7 @@ package Inline;
 
 use strict;
 require 5.006;
-$Inline::VERSION = '0.54';
+$Inline::VERSION = '0.54_01';
 $Inline::VERSION = eval $Inline::VERSION;
 
 use AutoLoader 'AUTOLOAD';
@@ -12,6 +12,7 @@ use Carp;
 use Cwd qw(abs_path cwd);
 use File::Spec;
 use File::Spec::Unix;
+use Fcntl qw(LOCK_EX LOCK_UN);
 
 my %CONFIG = ();
 my @DATA_OBJS = ();
-- 
1.9.1

